pom.xml

```diff
@@ -63,6 +63,8 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+            <version>4.13</version>
+            <scope>test</scope>
```

Comment:
`junit` is a managed dependency from the `com.jnape.palatable.parent` pom, where its scope is indeed specified as `test`, which is why you don't see it here.  So this scope is redundant.

Reply:
I'd probably rather upgrade the dependency in the parent and let it cascade down. I need to think about this change because to make sure there are no ramifications to the test-support types that the lambda `test-jar` offers for client libraries that use the older junit versions.

Reply:
Good catch; I'm going to do some more digging also.